doc/1.4/language.md

```diff
@@ -3254,6 +3259,40 @@ will give a compile error unless it appears in one of the following contexts:
   resulting object reference, the corresponding index variable of
   the object array will have an unknown value.
 
+### Method References as Function Pointers
+It is possible to retrieve a function pointer for a method by using `&` on a
```

Comment:
nitpick: maybe "the prefix operator `&`", to distinguish from the bitwise AND operator

Reply:
No "maybe" about it, that phrasing is just better.